sql_queries/Populate_SpatialSubsetTable_csiro_cmar_underway.sql

```diff
@@ -0,0 +1,43 @@
+SET SEARCH_PATH = marvl3, public;
+-- CSIRO UNDERWAY
+\echo 'CSIRO UNDERWAY'
+INSERT INTO marvl3.spatial_subset(
+source_id,
+origin_id,
+"LONGITUDE",
+"LONGITUDE_QC",
+"LATITUDE",
+"LATITUDE_QC",
+"TIME",
+"TIME_QC",
+"DEPTH",
+"DEPTH_QC",
+"TEMP",
+"TEMP_QC",
+"PSAL",
+"PSAL_QC",
+geom
+)
+SELECT 
+s.source_id,
+d."SURVEY_ID",
+d."LONGITUDE",
+'0',
```

Comment:
Again Lat, Lon and Time might not have QC info because they are assumed always good.